unit/test/agents.Tests.ps1

```diff
@@ -130,5 +130,26 @@ InModuleScope VSTeam {
             { Disable-VSTeamAgent -Pool 36 -Id 950 } | Should Throw
          }
       }
+
+      Context 'Update-VSTeamAgent by ID' {
+         Mock Invoke-RestMethod
+
+         It 'should update the agent with passed in Id' {
+            Update-VSTeamAgent -Pool 36 -Id 950
+
+            Assert-MockCalled Invoke-RestMethod -Exactly -Scope It -Times 1 -ParameterFilter {
+               $Method -eq 'Post' -and
+               $Uri -eq "https://dev.azure.com/test/_apis/distributedtask/pools/36/messages?api-version=$([VSTeamVersions]::DistributedTask)&agentId=950"
```

Comment:
It might be better to test each query parameter in its own line. Have a look here:
https://github.com/DarqueWarrior/vsteam/blob/89789e8a478f0a0c96404643a589d8ed35a61f34/unit/test/Remove-VSTeamWorkItem.Tests.ps1#L46

Reply:
great idea, thanks!

Reply:
thanks. Looks much better now :)